### PR TITLE
Typos from marvin.cs.uidaho.edu: V

### PR DESCRIPTION
**Pull request by skangas**
This replaces PR https://github.com/codespell-project/codespell/pull/1971. I've added in several variations myself.

See: http://marvin.cs.uidaho.edu/misspell.html

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -38553,12 +38568,19 @@ valies->values
 valif->valid
 valitdity->validity
 valkues->values
+vallay->valet
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:38571]: Perhaps `valley` (and plural forms) too?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -38632,7 +38657,11 @@ varient->variant
 varients->variants
 varierty->variety
 variey->variety
+varification->verification
+varifications->verifications
+varified->verified
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:38662]: `varifies->verifies`
`varifiying->verifiying`

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -38860,22 +38944,36 @@ vieports->viewports
...
 vigourous->vigorous
 vill->will
+villan->villain
+villans->villains
 villian->villain
+villians->villains
 villification->vilification
 villify->vilify
 villin->villi, villain, villein,
+vinal->vinyl
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:38959]: `vinal` appears to be an alternative spelling of `vinyl` as a synthetic textile fibre. Move to _rare_?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:38571]: Added.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:38662]: Added.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:38959]: Yes, done.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**